styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -1,3 +1,98 @@
+.rsHomepage-announcement {
+  display: flex;
+  margin-top: -70px;
+  width: 100%;
+}
+
+.rsHomepage-annLeft {
+  background-image: url('/imgs/home-pageAnn-left.png');
```

Comment:
This needs to be a relative path.  (e.g. `../imgs/home-pageAnn-left.png`)
Otherwise this will only ever work on zoolander.

Reply:
I'll change this but this placeholder is likely going to be overwritten in the general with an inline style tag. It was only supposed to work in zoolander.
ie:
```
<div class="rsHomepage-annLeft" style="background-image: url(the_asset_url);">
```

Reply:
Where is that asset gonna live?  To me it seems like home page changes warrant a code/qe review anyway due to visibility, so managing that in zoolander seems like the optimal solution here.